src/test/java/org/mule/tools/apikit/MainAppScaffolderTest.java

```diff
@@ -708,6 +761,35 @@ private void simpleGenerateWithCustomDomainWithMultipleLC() throws Exception {
     assertEquals(5, countOccurences(s, "<logger level=\"INFO\" message="));
   }
 
+  private void simpleGenerateWithCustomExternalDomainWithMultipleConfigs() throws Exception {
```

Comment:
@juandesi This test should verify that all the domain's listener configs are parsed and the final scaffold is correct. The MuleDomain is parsing the configs correctly, I'm not sure about the final verification of the generated config.